docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -64,6 +64,9 @@ Hotspot support (Web Auth Enforcement)::
   PacketFence can also be configured as hotspot, if you have a manageable device that supports an external captive portal (like
 Cisco WLC or Aruba IAP).
 
+RADIUS support (RADIUS Enforcement)::
+  PacketFence can act as RADIUS only server, without having a registration, isolation, portal interface. In case you want to use PacketFence for your 802.1X authentication only. 
```

Comment:
I'd rephrase that to be more "general", to be future proof.
"PacketFence can be used as a RADIUS server without having registration, isolation, portal features in the case only RADIUS authentication for network equipment is what matter."